charts/cluster-scanner/templates/_helpers.tpl

```diff
@@ -279,3 +289,54 @@ Return local registry secrets in the correct format: <namespace_name>/<secret_na
     */}}
     {{- .Values.global.sysdig.accessKeySecret | default "" -}}
 {{- end -}}
+
+{{- define "cluster-scanner.nats.tls.volumePath" -}}
+/opt/kubernetes_secrets/custom_cert
+{{- end -}}
+
+{{/*
+Return true if any of TLS CA, private Key and certificate have been provided.
+*/}}
+{{- define "cluster-scanner.nats.tls.customTLS" -}}
+    {{- if or (not (empty .Values.runtimeStatusIntegrator.natsJS.tls.key)) (not (empty .Values.runtimeStatusIntegrator.natsJS.tls.cert)) -}}
+        {{- true -}}
+    {{- else -}}
+        {{- false -}}
+    {{- end -}}
```

Comment:
```suggestion
    {{- or (not (empty .Values.runtimeStatusIntegrator.natsJS.tls.key)) (not (empty .Values.runtimeStatusIntegrator.natsJS.tls.cert)) -}}
```
shouldn't this be equivalent?

Reply:
Unfortunately, it is not. Each definition is treated as a string, and the `"false"` string would evaluate to `true`, see my comment https://github.com/sysdiglabs/charts/pull/1366#discussion_r1345803757.

I'll just simplify this condition, and make it return an empty string if it evaluates to `false`.